src/codeu/chat/DerbyStore.java

```diff
@@ -110,7 +110,7 @@ public User getUser(Uuid userid) throws SQLException, IOException {
 		ResultSet user = getUser.executeQuery();
 		
 		if (user.next()) {
-			return new User(Uuid.parse(user.getString("userid")), user.getString("name"), Time.fromMs(user.getLong("creation")));
+			return new User(Uuid.parse(user.getString("userid")), user.getString("name"), user.getString("password"), Time.fromMs(user.getLong("creation")));
```

Comment:
Eh, then you'd be back to using magic indexes that you have to make sure stay synced. I'd probably just use String variables. Maybe encapsulate them in a separate class if you don't want them to clutter up your code.